server/src/main/java/io/spine/server/projection/ProjectionRepository.java

```diff
@@ -84,19 +84,20 @@ static Timestamp nullToDefault(@Nullable Timestamp timestamp) {
     }
 
     /** Obtains {@link EventStore} from which to get events during catch-up. */
-    EventStore getEventStore() {
-        return getBoundedContext().getEventBus()
-                                  .getEventStore();
+    EventStore eventStore() {
+        return this.boundedContext()
```

Comment:
Not sure why we need `this.`. We don't refer to the instance methods anywhere else like this.